<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringBuilder::WidgetOnlyPositionFixer</class>
 <widget class="QWidget" name="StringBuilder::WidgetOnlyPositionFixer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>98</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="StringBuilder::WidgetPositionFixer" name="widgetPositionFixer">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringBuilder::WidgetPositionFixer</class>
   <extends>QFrame</extends>
   <header>stringbuilder/widgets/widgetpositionfixer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
